js/PixelPainter.js

```diff
@@ -0,0 +1,97 @@
+const colours = ['#000000', '#FF0000', '#FFA500', '#008000', '#0000FF', '#800080', '#FFC0CB', '#808080', '#FA8072', '#FFD700', '#90EE90', '#00FFFF', '#ADD8E6', '#E6E6FA', '#FFE4E1'];
```

Comment:
It's best to wrap everything in a closure so that all your variables are private and can't be edited by anything outside the closure.